### ISSUE TITLE
Switch to base-pom

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis-verifier#51]
Like for syndesis-rest. Also important for aligning for a monorepo setup.

Would this be hard, @zregvart ? // cc @iocanel

**Comment by zregvart** [comment]
> Would this be hard, @zregvart ? // cc @iocanel

Not so hard I think, perhaps we could also have a standalone syndesis-partent POM for rest, verifier and connectors?

**Comment by rhuss** [comment]
yes, which we will have anyway when moving to syndesisio/syndesis ....
